README.md

```diff
@@ -39,31 +96,42 @@ You can pass additional arguments to Theia after the image name, for example to
 
     # Linux, macOS, or PowerShell
     docker run -it --init -p 3000:3000 --expose 9229 -p 9229:9229 -v "$(pwd):/home/project:cached" theiaide/theia:next --inspect=0.0.0.0:9229
-    
+
     # Windows (cmd.exe)
     docker run -it --init -p 3000:3000 --expose 9229 -p 9229:9229 -v "%cd%:/home/project:cached" theiaide/theia:next --inspect=0.0.0.0:9229
 
-### Image Variants
+### Electron Apps
 
-#### `theiaide/theia:latest`
+<img
+    src='./assets/electron.png'
+    alt='electron logo'
+    width="50px"
+/>
 
-This image is based on the latest stable released version.
+| Application Name | Description | Documentation |
+|:---|:---|:---|
+| `theia-cpp-electron` | Theia-based C/C++ desktop IDE | [docs](./theia-cpp-electron/README.md) |
+| `theia-electron` | Theia-based JavaScript/TypeScript (Web Technologies) desktop IDE | [docs](./theia-electron/README.md) |
 
-#### `theiaide/theia:<version>`
+---
 
-This image is based on the given stable released version.
+<br />
 
-#### `theiaide/theia:next`
+## Plugins
 
-This image is based on the nightly published version.
+Theia supports VS Code extensions as plugins. If a given image has plugin support, one can
+include their own plugins under the user's home: `/home/theia/plugins`.
 
-#### `theiaide/theia:<version>-next.<commit hash>`
+## Tips & Tricks
 
-This image is based on the given nightly published version.
+- #### Build Options:
+    - `--build-arg strip=true` strips the application to save space but with reduced debuggability.
+    - `--init` injects an instance of [tini](https://github.com/krallin/tini) in the container, that will wait-for and reap terminated processes, to avoid leaking PIDs.
+    - `--security-opt seccomp=unconfined` enables running without [the default seccomp profile](https://docs.docker.com/engine/security/seccomp/) for debugging.
```

Comment:
This is debugger-specific, with many not requiring this at all. We used to need it for C/C++ debugging but no longer with more recent Docker (we can keep for a while since not everyonoe will use latest docker). Seems to also be used for go. Rust is the same as  C/C++.

Reply:
I think it would be good to add something about the frequent permission issues people face using docker volumes. Not sure if here would be the best place. If you have an idea, go ahead. Else I don't want to delay merging this PR further so we can take care in a follow-up.